bioptim/dynamics/configure_problem.py

```diff
@@ -105,6 +112,10 @@ def torque_driven(ocp, nlp, with_contact: bool = False, fatigue: FatigueList = N
             A reference to the phase
         with_contact: bool
             If the dynamic with contact should be used
+        implicit_dynamics: bool
+            If the implicit dynamic should be used
```

Comment:
This PR seems to introduce implicit_dynamics as well.. Since they are now already merged, please make this a single PR